custom/icds/templates/icds/messaging/indicators/hin/aww_vhnd_survey.txt

```diff
@@ -0,0 +1 @@
+रिकार्ड के जानकारी के आधार पर आापके केन्द्र में पूर्व माह में VHSND नहीं आयोजित किया गया है। कृपया प्रत्येक माह नियमित रूप से VHSND का आयोजन सुनिश्चित करें।
```

Comment:
for the hindi and telugu translations, were the template files saved with utf-8 encoding?

i've also been adding a note at the top of each one (example [here](https://github.com/dimagi/commcare-hq/blob/744e8647c4aeb1260f2ae61cb328c433f8f3e435/custom/icds/templates/icds/messaging/indicators/hin/aww_aggregate_performance.txt)) to help make it clear to anyone else editing these templates later on.

Reply:
Clearly I wasn't paying enough attention when I copied those templates, since the notes were in the ones I copied (and then replaced the content). https://github.com/dimagi/commcare-hq/pull/21504/commits/eac59651290dc97fade151fca2f7cd3dd9335bb9